### PR TITLE
Orb 540 - M.8

### PR DESCRIPTION
**Pull request by luduvigo**

### CODE REVIEW

**Comment by luduvigo** [comment]
Can you please update this PR and use SafeMath where is missing?

File: contracts/modules/AutoCompoundModule.sol [hunk 1]
```diff
@@ -68,8 +71,18 @@ contract AutoCompoundModule is BaseModule {
...
             )
         ).slot0();
+
         //returns true if the value of uncollected fees * 100 is greater than amount in the position * threshold
-        return (((uncollectedFees0 * sqrtPriceX96) / 2**96 + (uncollectedFees1 * 2**96) / sqrtPriceX96) * 100 >
-            ((amount0 * sqrtPriceX96) / 2**96 + (amount1 * 2**96) / sqrtPriceX96) * feesThreshold);
+        // (((tick - tickDelta) / tickSpacing) * tickSpacing, ((tick + tickDelta) / tickSpacing) * tickSpacing);
+        return ((
+            (uncollectedFees0.mul(uint256(sqrtPriceX96)))
+                .div(uint256(2**96).add(uncollectedFees1.mul(uint256(2**96))).div(uint256(sqrtPriceX96)))
```
Comment (RiccardoGalbusera) [inline contracts/modules/AutoCompoundModule.sol:79]: Careful, this is not the same operation as before! Also, the cast uint256(2**96) is really needed?

File: contracts/modules/IdleLiquidityModule.sol [hunk 2]
```diff
@@ -3,6 +3,7 @@
 pragma solidity 0.7.6;
 pragma abicoder v2;
 
+import '@openzeppelin/contracts/math/SafeMath.sol';
```
Comment (RiccardoGalbusera) [inline contracts/modules/IdleLiquidityModule.sol:6]: please remove this import if package isn't used

**Review by RiccardoGalbusera — CHANGES_REQUESTED** [review]

File: contracts/modules/AutoCompoundModule.sol (hunk 1 above)
Comment (RiccardoGalbusera) [reply contracts/modules/AutoCompoundModule.sol:79]: and please remove the comment at line 76

**Pull request merged**